test/tst_standardize.jl

```diff
@@ -1,9 +1,8 @@
 X = collect(Float64, reshape(1:40, 10, 4))
 x = rand(10) * 10
 
-D = DataFrame(A=rand(10), B=collect(1:10), C=[hex(x) for x in 11:20])
-D_NA = deepcopy(D)
-allowmissing!(D_NA, :A)
+D = DataFrame(A=rand(10), B=1:10, C=string.(11:20; base=16))
+D_NA = allowmissing!(copy(D))
```

Comment:
did the effects of `copy` and `deepcopy` change for DataFrames? I don't remember why we used `deepcopy` in the first place but I am guessing we tried `copy` first. Any guesses?

Reply:
I can never actually recall the precise effects of `copy` vs `deepcopy`, for DataFrames,
but I know that even if `copy` shares column vectors, it does not matter,
as `allowmissing!` allocates new column vectors anyway with a different type. `Union{T. Missing}`.